src/main/resources/inspectionDescriptions/MissingTerminals.html

```diff
@@ -0,0 +1,18 @@
+<html>
+<body>
+<p>Detects missing terminals
+    <!--TODO: Write a better description? -->
```

Comment:
Something like this might work (feel free to adjust it):
> Terminals are static strings or regexes used in the grammar. They should be declared in a match block.

Reply:
Now that a description has been added, the TODO and template comments should be removed.